src/QsCompiler/DataStructures/ReservedKeywords.fs

```diff
@@ -192,14 +192,16 @@ module InternalUse =
     let UnitArgument = "__unitArg__"
 
     /// contains all keywords reserved due to clashes with auto-generated Q# code, or generated C# code
-    let CsKeywords = 
-        let reserved = [
+    let CsKeywords = ImmutableHashSet.CreateRange [
 
             "Allocate"; 
             "Release"; 
             "Borrow"; 
             "Return";
 
+            "Data";
+            "Item";
```

Comment:
So these additions resolve microsoft/qsharp-runtime#50 and microsoft/qsharp-runtime#37 by forbidding the identifiers `Data` and `Item` entirely? That seems a bit extreme, but just my 2¢. In any case, if we decide to resolve those issues by forbidding affected identifiers, we need `Item1`, `Item2`, etc. instead of just `Item`, I think.

Reply:
It wasn't intended to resolve anything, but merely to at least give a more comprehensive error until we address it (all of the things listed here as clashing with C# should really be usable - so I figured I might as well add them).